fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/node/LongBindMarkerNode.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.node;
+
+import com.ibm.fhir.database.utils.query.expression.BindMarkerNodeVisitor;
+
+/**
+ * A bind marker representing a Long value
+ */
+public class LongBindMarkerNode extends BindMarkerNode {
+    // The Long value (can be null)
```

Comment:
I've seen a couple  of these  markers... 
unpacking this comment,  is there a spot where javadoc could signal that the value could be null...

Reply:
I mentioned nullable in the class javadoc